exercises/verify.cpp

```diff
@@ -7,12 +7,33 @@ The number 3 is [not] present in the array.
 */
 
 #include <iostream>
+#define arraySize 10
```

Comment:
le costanti si scrivono in maiuscolo, quindi `ARRAY_SIZE`

fonte: https://en.wikipedia.org/wiki/Naming_convention_(programming)#Examples_of_multiple-word_identifier_formats

![image](https://user-images.githubusercontent.com/519778/168091634-5982954b-a3c9-46e5-b53f-ba152e57314c.png)